### ISSUE TITLE
Remove back button for error screens

### ISSUE THREAD

**Issue by phavekes** [OpenConext/OpenConext-engineblock#1551]
[This issue is imported from pivotal - Originaly created at Mar 20, 2019 by Pieter van der Meulen](https://www.pivotaltracker.com/story/show/164757267)

TL/DR: Remove the back button from all the engine error screens because it likely to fail with another error and there is no working alternative.

What the button does is go back two pages in the browsing history. While this may work in some situations, in most it won\'t and will lead to another error. The idea of the back button is that the user is brought back to the SP there where she tried to login.

The proper SAML way would be to send a SAMLResponse with StatusCode other than Success to the ACS Location of the SP where the authentication was started, but this only works if we know which SP that was, which for this specific error is unlikely. It also requires the SP to be able to handle such a SAMLResponse gracefully, and we know that many SPs currently don\'t do that. Such functionally merits its own issue to be discussed there further.

We are aware that removing the button has the effect of leaving the user stranded at an engine error page, but this is currently the best we can do. We are making the page as nice and a useful to the user as we can (#164121204)

**Comment by phavekes** [comment]
TL/DR: Remove the back button from all the engine error screens because it likely to fail with another error and there is no working alternative.

What the button does is go back two pages in the browsing history. While this may work in some situations, in most it won\'t and will lead to another error. The idea of the back button is that the user is brought back to the SP there where she tried to login.

The proper SAML way would be to send a SAMLResponse with StatusCode other than Success to the ACS Location of the SP where the authentication was started, but this only works if we know which SP that was, which for this specific error is unlikely. It also requires the SP to be able to handle such a SAMLResponse gracefully, and we know that many SPs currently don\'t do that. Such functionally merits its own issue to be discussed there further.

We are aware that removing the button has the effect of leaving the user stranded at an engine error page, but this is currently the best we can do. We are making the page as nice and a useful to the user as we can (https://www.pivotaltracker.com/story/show/164121204) (Pieter van der Meulen - Mar 20, 2019)

**Comment by phavekes** [comment]
I took the liberty to move your comment to the story description. (Michiel Kodde - Mar 20, 2019)

**Comment by phavekes** [comment]
@pmeulen how does this story relate to: #164243746? (Michiel Kodde - Mar 20, 2019)

**Comment by phavekes** [comment]
@michielkodde 

> I took the liberty to move your comment to the story description.

Yes, thx, that is where I intended to put it in the first place.

>  how does this story relate to: #164243746?

Both deal with the back button and the browsing history and trying to make that behave in a useful way when a SAML Web-SSO proxy is involved. (Pieter van der Meulen - Mar 20, 2019)

**Comment by phavekes** [comment]
Back buttons have been removed in this [PR](https://github.com/OpenConext/OpenConext-engineblock/pull/670) (Michiel Kodde - May 2, 2019)
